src/ast/passes/parser.h

```diff
@@ -59,6 +59,7 @@ inline std::vector<Pass> AllParsePasses(
   passes.emplace_back(CreateMapSugarPass());
   passes.emplace_back(CreateNamedParamsPass());
   passes.emplace_back(CreatePidFilterPass());
+  passes.emplace_back(CreateResolveStatementImportsPass());
```

Comment:
Just add an explanation here to distinguish from the above, and why this comes later?